index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+        <header class="navbar">
+        <div class="container">
+            <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+</header>
+    <header class="header">
+        <div class="container">
+                    <div class="textInfo">
+             <h1>I’m Mariane
+                Glad to see you!</h1>
+                
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>        
+    </div>
+    <button class="con">LET'S CONNECT </button>
```

Comment:
As highlighted by the previous reviewer, I suggest you replace this second `<header>` element with a `<section>`. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info